.circleci/config.yml

```diff
@@ -34,7 +34,16 @@ jobs:
           command: PATH="venv/bin:$PATH" yamllint -c .yamllint.yaml .
       - run:
           name: PyTest with linters
-          command: PATH="venv/bin:$PATH" script/entrypoint
+          # yamllint disable rule:line-length
+          command: |
+            if [ -n "$CIRCLE_PR_NUMBER" ]; then
+              echo "Cannot pass creds to forked PRs," \
+                "so skipping routine and SQL tests"
+              PATH="venv/bin:$PATH" script/entrypoint -m "not (routine or sql or integration or java)"
+            else
+              PATH="venv/bin:$PATH" script/entrypoint
+            fi
+          # yamllint enable rule:line-length
```

Comment:
I figured out the problem: we run integration tests in the `bigquery-etl-integration-test` project which does not have permissions to access the `moz-fx-data-circleci-tests-bigquery-etl` storage bucket. The easiest workaround for now is to check for forks in the `build` step and only run SQL/routine tests for non-forks. This way `entrypoint` and how we run tests doesn't need to change.